<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetPassengers">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>260</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassengers">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>11</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Passengers</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>listWidgetPassengers</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddPassenger">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>40</y>
      <width>40</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemovePassenger">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>270</y>
      <width>40</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leditPassport">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetFlights">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>761</width>
      <height>521</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBottonAddNewFlight">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>500</y>
      <width>310</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Flight</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionChoose"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose">
   <property name="text">
    <string>Choose...</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
